protocol/contracts/v2/strategies/GnosisStrategy.sol

```diff
@@ -0,0 +1,149 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (v2/strategies/GnosisStrategy.sol)
+
+import { SafeERC20, IERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { AbstractStrategy } from "contracts/v2/strategies/AbstractStrategy.sol";
+import { CommonEventsAndErrors } from "contracts/common/CommonEventsAndErrors.sol";
+
+contract GnosisStrategy is AbstractStrategy {
+    using SafeERC20 for IERC20;
+    string public constant VERSION = "1.0.0";
+
+    /**
+     * @notice The underlying gnosis safe wallet which is reported on.
+     */
+    address public gnosisSafeWallet;
+
+    /**
+     * @notice The list of assets which are reporting on for equity performance updates
+     * Total balances = the gnosis safe balance of the asset + any delta from `assetBalanceDeltas`
+     */
+    address[] public assets;
+
+    event AssetsSet(address[] _assets);
+    event Borrow(uint256 amount);
+    event Repay(uint256 amount);
+
+    constructor(
+        address _initialRescuer,
+        address _initialExecutor,
+        string memory _strategyName,
+        address _treasuryReservesVault,
+        address _gnosisSafeWallet
+    ) AbstractStrategy(_initialRescuer, _initialExecutor, _strategyName, _treasuryReservesVault) {
+        gnosisSafeWallet = _gnosisSafeWallet;
```

Comment:
I see no setters, make immutable?

Reply:
Good spot, thanks

Reply:
Fixed in https://github.com/TempleDAO/temple/pull/814/commits/2cf3760a6d25816f1b2ea81911375af2c6f4914b